packs/feat-effects/stance-air-shroud.json

```
@@ -0,0 +1,36 @@
+{
+    "_id": "XFc3dVzTe7KnpjuP",
+    "img": "icons/magic/air/air-wave-gust-blue.webp",
+    "name": "Stance: Air Shroud",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Air Shroud]<br /><br />Turbulent air suffuses your kinetic aura. This doesn't displace other matter but provides air for creatures in your kinetic aura that need air to breathe. Attacks with ranged weapons take a -1 circumstance penalty to their attack rolls if their paths pass through your kinetic aura. Air in your kinetic aura is difficult terrain for flying creatures other than yourself.</p>"
+        },
+        "duration": {
+            "expiry": null,
+            "sustained": false,
+            "unit": "encounter",
+            "value": -1
+        },
+        "level": {
+            "value": 1
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": ""
+        },
```

```suggestion
        "publication": {
            "license": "OGL",
            "remaster": true,
            "title": "Pathfinder Rage of Elements"
        },
```
Kind of a pain because there's no UI for this in Foundry currently but we want source info on effects too.